test/integration/client-side-encryption/driver.test.ts

```diff
@@ -105,10 +115,12 @@ describe('Client Side Encryption Functional', function () {
   });
 
   describe('BSON Options', function () {
+    let client: MongoClient;
+
     beforeEach(function () {
-      this.client = this.configuration.newClient();
+      client = this.configuration.newClient();
 
-      const noop = () => {};
+      const noop = () => null;
```

Comment:
Our linter complains about empty functions, rather than disable the rule, we've used `return null` in a few places, it does have different meaning, we should keep in mind that this means the function will return something different from before. But it has served us to offer meaning that the result value is "omitted".

Reply:
Sounds good!